### ISSUE TITLE
Edit the AddToCart end point to remove items from your cart.

### ISSUE THREAD

**Issue by Palisar** [Code2Gether-Discord/Congo#93]

**Comment by Palisar** [comment]
This is self explanatory the CartController.cs needs to be updated with a method for handling this.
Also dont forget we need mediatr here so you will have to add a command to the CartCommands.cs and create a handler for this.
Feel free to ask any questions with this issue.

**Comment by valincius** [comment]
I think we can reuse the add to cart action. Rename it like “update cart” then if the quantity is 0, remove the item

**Comment by Palisar** [comment]
I was thinking that at first also but this handler already handles creates and updates if we want to keep the code simple and readable would it be better to give it it's own hander ?

**Comment by pobiega** [comment]
I agree with John, keep it as one and rename it. It really won't be complicated or unreadable, its just two extra lines of code or so :)

**Comment by Palisar** [comment]
Ok got it !
